extend/generic-writer/configuration/configuration.md

```diff
@@ -0,0 +1,718 @@
+- `BINARY-GZ` - input is sent as gzipped binary data. See [example](https://bitbucket.org/kds_consulting_team/kds-team.wr-generic/src/master/tests/functional/binary_gz/)
+- `EMPTY_REQUEST` - sends just empty requests. Usefull for triggerring webhooks, DELETE calls, etc. 
+As many requests as there are rows on the input are sent. Useful with `iterate_by_columns` enabled to trigger multiple endpoints. 
+See [example 022](https://bitbucket.org/kds_consulting_team/kds-team.wr-generic/src/master/docs/examples/022-empty-request-iterations-delete/) 
+
+
+```json
+        "request_options": {
+            ...,
+            "content": {
+                "content_type": "JSON",
+            ....
+```
+
+#### JSON Mapping
+
+[REQUIRED for JSON based content type] This section defines the CSV 2 JSON conversion in case of JSON content type. 
+
+##### Nesting delimiter
+
+[REQUIRED] A string that is used for nesting. e.g. `__`. This way you can define nested objects based on column names.
+
+e.g. When set to `__` a column value `address__streed` will be converted to `{"address"{"street":"COLUMN_VALUE"}}`
+
+```json
+"request_options": {
+                           ...,
+            "content": {
+                "content_type": "JSON",
+                "json_mapping": {
+                    "nesting_delimiter": "_",
+                    ...
+```
+
+See example [008](https://bitbucket.org/kds_consulting_team/kds-team.wr-generic/src/master/docs/examples/008-simple-json-nested-object-delimiter/)
+
+
+##### Chunk size
+[REQUIRED] Defines how many rows are being sent in a single request. When set to `1` a single object is sent `{}` (see [example 002](https://bitbucket.org/kds_consulting_team/kds-team.wr-generic/src/master/docs/examples/002-simple-json-chunked-single/)), 
+when set to >1 an array of objects is sent `[{}, {}]` (see [example 003](https://bitbucket.org/kds_consulting_team/kds-team.wr-generic/src/master/docs/examples/003-simple-json-chunked-multi/))
+
+```json
+"request_options": {
+            ...,
+            "content": {
+                "content_type": "JSON",
+                "json_mapping": {
+                    "nesting_delimiter": "_",
+                    "chunk_size": 1,
+```
+
+##### Column datatypes
+
+
+Optional configuration of column types. This version supports nesting (three levels) and three datatypes:
+
+- `bool` -  Boolean value  case-insensitive conversion: `t`, `true`, `yes`, `1`,`"1"` to `True` and `f`, `false`, `no` to `False`
+- `string` - String
+- `number` - Number
+- `object` - Object - valid JSON array or JSON object, e.g. ["1","2"], {"key":"val"}
+
+###### Autodetect
+
+[OPTIONAL] Default value `true
+`
+Set this option to `true` to make the parser automatically detect the above datatypes. It may be used in combination with 
+`datatype_override` option to force datatype to some columns.
+
+###### Column datatype override
```

Comment:
yeah - this is hell of a nesting, the whole page should be split in at least 3-4 pages (following the main nodes of the configuration), so you don't need to use h6+